.gitignore

```diff
@@ -1,23 +1,31 @@
-# Compiled class file
-*.class
-
 # Log file
 *.log
 
 # BlueJ files
 *.ctxt
 
+*.class
+.classpath
+.settings
+.project
+*.iml
+*.iws
+*.ipr
+.idea
+
 # Mobile Tools for Java (J2ME)
 .mtj.tmp/
 
 # Package Files #
 *.jar
 *.war
-*.nar
 *.ear
+*.nar
 *.zip
 *.tar.gz
 *.rar
 
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+# Ignore everything in this directory
+target
```

Comment:
End of Line

Reply:
Fixed in https://github.com/wso2-extensions/carbon-crypto-provider-hsm/pull/5